docs/next/guides/getting-started/setting-options.md

```diff
@@ -14,319 +14,219 @@ tags:
-  cells(row, col, prop) {
-    const cellProperties = {};
-
-    if (row === 0 && col === 0) {
-      cellProperties.readOnly = true;
+  data: Handsontable.helper.createSpreadsheetData(5, 10),
+  cells(row, column) {
+    if (row === 1 || row === 4) {
+      return { readOnly: true };
     }
-
-    return cellProperties;
-  }
-})
+  },
+  width: 'auto',
+  height: 'auto',
+  rowHeaders: true,
+  colHeaders: true,
+  licenseKey: 'non-commercial-and-evaluation'
+});
 ```
+:::
 
-## The cascading configuration
-
-Handsontable utilizes cascading configuration, which is a fast way to provide configuration options for the entire grid, along with its columns and particular cells.
+### Single cell
 
-Consider the following example:
+And finally, let's set the read-only only for one cell.
 
+::: example #example4 --html 1 --js 2
+```html
+<div id="example4" class="hot"></div>
+```
 ```js
-const container = document.querySelector('#example');
+const container = document.querySelector('#example4');
 
 const hot = new Handsontable(container, {
-  readOnly: true,
-  columns: [
-    {readOnly: false},
-    {},
-    {}
-  ],
-  cells(row, col, prop) {
-    const cellProperties = {};
-
-    if (row === 0 && col === 0) {
-      cellProperties.readOnly = true;
+  data: Handsontable.helper.createSpreadsheetData(5, 10),
+  cells(row, column) {
+    if (row === 1 && column === 1) {
+      return { readOnly: true };
     }
-
-    return cellProperties;
-  }
+  },
+  width: 'auto',
+  height: 'auto',
+  rowHeaders: true,
+  colHeaders: true,
+  licenseKey: 'non-commercial-and-evaluation'
 });
 ```
+:::
 
-The above notation will result in all TDs being read only, except for first column TDs which will be editable, except for the TD in top left corner which will still be read-only.
-
-## The cascading configuration model
+## `cells` vs `cell`
 
-The cascading configuration model is based on prototypal inheritance.
+In the Options section of the API, you can find two similar options, [cell](@/api/metaSchema.md#cells-2) and [cell](@/api/metaSchema.md#cell). Using one of these options (or both), you can associate the custom properties or overwrite any entire table or column option for a particular cell. Both options allow you to set initial values for specific cells. You can check if the values are propagated by calling the [getCellMeta](@/api/core.md#getcellmeta) method. The method returns an object with all built-in options as well as those added by the developer.
 
-### Constructor
+#### `cell`
 
-Configuration options that are provided using first-level and `updateSettings` method.
+The [cell](@/api/metaSchema.md#cell) option works great for cases when you have to set initial values for your custom properties or change the initial values for the build-in options. Once the changes are propagated to the cells' meta-objects, they can be modified by the Handsontable while it is running. For example, the cell read-only state can be toggled by the context menu.
```

Comment:
build-in options > built-in option

 while it is running > in the runtime?

toggled? I'm not sure about this one. >> altered? modified? changed?